src/main/java/fr/sncf/osrd/railjson/schema/rollingstock/RJSRollingStock.java

```diff
@@ -4,126 +4,213 @@
-    public double length;
+    @Json(name = "sub_type")
+    public String subType = null;
 
-    /** The mass of the train, in kilograms. */
-    public double mass;
+    public String series = null;
+
+    @Json(name = "sub_series")
+    public String subSeries = null;
+
+    public String variant = null;
+
+    @Json(name = "units_count")
+    public int unitsCount = -1;
 
     /**
-     * Inertia coefficient.
-     * The mass alone isn't sufficient to compute accelerations, as the wheels and internals
-     * also need force to get spinning. This coefficient can be used to account for the difference.
-     * It's without unit: effective mass = mass * inertia coefficient
+     * <p>Engineers measured a number of effort curves for each rolling stock. 
+     * These are referenced from effort curve profiles.
+     * Effort curves associate a speed to a traction force.
+     * https://en.wikipedia.org/wiki/Tractive_force#Tractive_effort_curves</p>
+     *
+     * <p>The type of this field is somehow misleading, each effort curve is an array
+     * of (speed, max_effort) tuples.</p>
      */
-    @Json(name = "inertia_coefficient")
-    public double inertiaCoefficient;
+    @Json(name = "effort_curves")
+    public Map<String, float[][]> effortCurves;
+
+    /**
+     * An effort curve profile is a set of rules which associate environmental conditions,
+     * such as line voltage or air conditionning settings, to effort curves
+     */
+    @Json(name = "effort_curve_profiles")
+    public Map<String, RJSEffortCurvesProfile[]> effortCurvesProfiles;
+
+    /**
+     * Each rolling resistance profile is a set of rules which associate environmental conditions,
+     * such as the load, to a rolling resistance.
+     */
+    @Json(name = "rolling_resistance_profiles")
+    public Map<String, RJSRollingResistanceProfile[]> rollingResistanceProfiles;
 
-    /** The rolling resistance force formula */
-    @Json(name = "rolling_resistance")
-    public RJSRollingResistance rollingResistance;
+    public RJSLivery[] liveries = null;
 
-    /** The list of capabilities (protection systems, signaling equipment) the train is able to deal with */
-    public RJSTrainCapability[] capabilities;
+    /** the length of the train, in meters.*/
+    public double length = Double.NaN;
 
     /** The max speed of the train, in meters per seconds. */
     @Json(name = "max_speed")
-    public double maxSpeed;
+    public double maxSpeed = Double.NaN;
 
     /**
      * The time the train takes to start up, in seconds.
      * During this time, the train's maximum acceleration is limited.
      */
     @Json(name = "startup_time")
-    public double startUpTime;
+    public double startUpTime = Double.NaN;
 
     /** The acceleration to apply during the startup state. */
-    @Json(name = "startup_acceleration")
-    public double startUpAcceleration;
+    @Json(name = "start_up_acceleration")
+    public double startUpAcceleration = Double.NaN;
```

Comment:
startup is a single word, written either start-up or startup. as using `-` in field names is bad practice, I'd rather use `start_up_acceleration` for the json and `startupAcceleration` for java, just like before